.github/workflows/ci-ubuntu-combined.yml

```diff
@@ -86,7 +86,7 @@ jobs:
           wget https://www.idris-lang.org/idris2-src/idris2-$IDRIS2_VERSION.tgz
           tar zxvf idris2-$IDRIS2_VERSION.tgz
           cd Idris2-$IDRIS2_VERSION
-          make bootstrap-build
+          make bootstrap
```

Comment:
Why this unrelated change?

Reply:
> Why this unrelated change?

This change was suggested by @fabianhjr 

https://github.com/idris-lang/Idris2/pull/1607#issuecomment-867676641

Reply:
So if I'm reading this correctly: https://github.com/idris-lang/Idris2/commit/b81b390f20f17f6beddf0c0a14f2463eafe0ceb0
all of this time we've been running the wrong command & it would not actually test anything?

Reply:
bootstrap-build was renamed bootstrap between v0.3.0 and v0.4.0